### PR TITLE
feature Counter-Rotating Forward Flight 

### PR DESCRIPTION
**Pull request by DaniloSchioppa**

### CODE REVIEW

**Comment by DaniloSchioppa** [comment]
This function calculates the various power components required for the flight of an helicopter, with counter rotating coaxial configuration, based on the provided input parameters.

The outputs include:
- Total Power Required (P_old): the sum of all power components necessary to sustain flight;
- Induced Power (P_i_old): the power contribution required to generate lift, accounting for the induced velocity created by the rotor during flight;
- Profile Power (P_0_old): the power expended to overcome the aerodynamic drag of the rotor blades;
- Fuselage Power (P_fus_old): the power needed to counteract the aerodynamic drag of the vehicle fuselage.

The required inputs for these calculations are:
- Flight speed (V_inf): the airspeed relative to the vehicle;
- Number of rotors (N_contra): the number of rotors installed on the vehicle;
- Rotor radius (R): the length from the center to the tip of the rotor blades;
- Vehicle mass (M): the total weight of the aircraft;
- Hover induced velocity (w_h): the airflow velocity induced by the rotors during hover;
- Altitude (height): the height above sea level, used to determine air density;
- Equivalent area (f): a parameter representing the vehicle’s total drag area;
- Angular velocity (Omega): the rotational speed of the rotors;
- Hover induced power coefficient (Pci_h): defined as Tc^(3/2)/rad(2) ;
- Hover profile power coefficient (Pc0_h): defined as  (σ Cd )/ 8 .

File: ContraFW.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    
+    # Calculation of the rotor disk area
+    A = pi * R**2
+
+    # Function to obtain air density based on altitude (simple approximation)
+    def air_density_at_altitude(h):
+        if h <= 11000:  # Troposphere
+            return 1.225 * (1 - 0.0000225577 * h)**4.2561
+        else:
+            return 0.36391 * np.exp(-0.0001577 * (h - 11000))
```
Comment (EttoreSaetta) [inline ContraFW.py]: Use the `ambiance` library for the ISA calculations

File: ContraFW.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            P_0_old.append(None)
+            P_fus_old.append(None)
+
+    return P_old, P_i_old, P_0_old, P_fus_old
+
+# Printing results for verification
+print("Total Required Power (P_old):", P_old)
+print("Induced Power (P_i_old):", P_i_old)
+print("Profile Power (P_0_old):", P_0_old)
+print("Fuselage Power (P_fus_old):", P_fus_old)
```
Comment (EttoreSaetta) [inline ContraFW.py]: Remove `print()` in the function

**Review by EttoreSaetta — CHANGES_REQUESTED** [review]
Solve the following points:

1. header at the beginning of the .py file is missing.
2. Test Cases missing.
3. Follow the comments below:

**Review by EttoreSaetta — APPROVED** [review]

**Pull request merged**